CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ scheme are considered to be bugs.
 
 * [#467](https://github.com/intridea/hashie/pull/467): Fixed `DeepMerge#deep_merge` mutating nested values within the receiver - [@michaelherold](https://github.com/michaelherold).
 * [#505](https://github.com/hashie/hashie/pull/505): Ensure that `Hashie::Array`s are not deconverted within `Hashie::Mash`es to make `Mash#dig` work properly - [@michaelherold](https://github.com/michaelherold).
+* [#508](https://github.com/hashie/hashie/pull/508): Fixed a bug in `Mash.load` that tried to use Mash#except which is a rails only method. - [@bobbymcwho](https://github.com/bobbymcwho).
```

Comment:
CHANGELOG line needs fixing, no period notably inside the sentence

Write "Fixed `Mash.load` no longer uses Rails-only `except`"

Reply:
![image](https://media.giphy.com/media/ZgqJGwh2tLj5C/giphy.gif)